### ISSUE TITLE
I need to create a Select2 field

### ISSUE THREAD

**Issue by armafile** [tabalinas/jsgrid#1151]
I need to create a single Select2 field in my JSGrid like this https://jsfiddle.net/NathanFriend/rLmztr2d/ but not a multiple select, just filter the items to choose one. I got some examples, but it is not what I need. Can You help me? 
![image](https://user-images.githubusercontent.com/41165011/42649455-260fd3c4-85e0-11e8-84cd-a7194ac0616f.png)

**Comment by lnpg87** [comment]
it works great

**Comment by vignesh-locus** [comment]
How to pre-populate select2 in the grid.

**Comment by uqix** [comment]
@vignesh-locus you can specify select options by Field.items

**Comment by aaron-h2** [comment]
How can we turn this into a multiple select2?

**Comment by vignesh-locus** [comment]
> @vignesh-locus you can specify select options by Field.items

Yeah right. Actually I was facing the problem that if we repopulate or use multi-select then grid column css will collapse and select will look like basic HTML select. Then, we inserted the values along with set timeout which solves the problem completely.

**Comment by xupmtp** [comment]
I use multiple select2,but when i update item,it can only get the first value
